package/tests/example_pipelines.json

```diff
@@ -0,0 +1,95 @@
+{

```

Comment:
This JSON is the matching API response for the `example_catalog` and `example_pipelines` test fixtures.